inspec.yml

```diff
@@ -20,21 +17,26 @@ inputs:
   - name: administrators
     description: 'List of authorized users in the local Administrators group'
     type: Array
-    value: ["Administrator",
-            "Domain Admins",
-            "Enterprise Admins"]
+    sensitive: true
+    value:
+      - "Administrator"
+      - "Domain Admins"
+      - "Enterprise Admins"
 
   - name: administrators_domain
     description: 'List of authorized users in the local Administrators domain group'
     type: Array
-    value: ["Administrator",
-            "Domain Admins",
-            "Enterprise Admins"]
+    sensitive: true
+    value:
+      - "Administrator"
+      - "Domain Admins"
+      - "Enterprise Admins"
 
   - name: temp_account
     description: 'List of temporary accounts on the system'
     type: Array
-    value: ["tempuser"]
+    value:
+      - "tempuser"
 
   - name: emergency_account
```

Comment:
For some inputs like this, I had initially changed the value to initiate the array. However, a hyphen `-` seems to be declaring an element so I've changed these back to square brackets `[]` instead.